cfgov/unprocessed/css/organisms/mega-menu.less

```
@@ -365,6 +365,13 @@ body {
         margin-top: 0;
     }
 
+    // Reset default h2 heading styles.
+    h2 {
+        font-size: 1em;
+        line-height: 22px;
```

Can we make this unitless?

```suggestion
        line-height: 1.375;
```